common/SmokeTests/SmokeTest/Program.cs

```diff
@@ -13,15 +14,27 @@ class Program
         static async Task<int> Main(string[] args)
         {
             Console.WriteLine("SMOKE TEST FOR TRACK 2 LIBRARIES");
+            Console.WriteLine();
 
             await KeyVaultTest.RunTests();
             await BlobStorageTest.RunTests();
             await EventHubsTest.RunTests();
+            await EventHubsTest.Cleanup();
```

Comment:
Cleanup should happen within the `RunTests()` method so the interface is consistent.